telemetry/specs/INT.mdk

```diff
@@ -1141,9 +1174,11 @@ INT Metadata Header and Metadata Stack, followed by TCP payload:
     0                   1                   2                   3
     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   | Ver=1 | 0 |0|0|0|     Reserved      | HopML=2 |RemainingHopC=6|
+   | Ver=2 | 0 |0|0|0|0|   Reserved      | HopML=2 |RemainingHopC=6|
```

Comment:
I don't see any definition of a new flag bit after the "M" bit.
The only change here should be "Ver=1" --> "Ver=2".

Reply:
What Flag bit after M are you referring to?

Reply:
This change adds a fourth 0 before the Reserved field. That does not belong.